versions/3.0.md

```diff
@@ -979,56 +998,75 @@ Field Name | Type | Description
 
 Field Pattern | Type | Description
 ---|:---:|---
+<a name="representations"></a>`*` | [Schema Object](#schemaObject)<span>&#124;</span> [Reference Object](#referenceObject) | A schema describing the response value for a specific `Content-Type`
 <a name="responseExtensions"></a>^x- | Any | Allows extensions to the OpenAPI Schema. The field name MUST begin with `x-`, for example, `x-internal-id`. The value can be `null`, a primitive, an array or an object. See [Specification Extensions](#specificationExtensions) for further details.
 
+Representations may take the form of a wildcard (`*`) to designate any `Content-Type`, or a regular expression for matching a specific type
```

Comment:
The statement is unclear, especially since "representations" is not defined in this spec (in [Response Object](https://github.com/OAI/OpenAPI-Specification/blob/650000a84424b3a3c7ae8f375e0774210299daa3/versions/3.0.md#response-object) ).  To be more precise, use a Representations Object definition in the spec:

   representations  |   [ Representations Object ]   | specify a mapping between Content-Media types and their representation definitions

Please include an example with a regular expression